templates/cluster-template-ipv6.yaml

```diff
@@ -143,6 +143,14 @@ spec:
   replicas: ${CONTROL_PLANE_MACHINE_COUNT}
   version: ${KUBERNETES_VERSION}
 ---
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
```

Comment:
this template is auto generated from kustomization.yaml in flavors/ipv6, you'll need to add that resource to it instead of directly modifying the generated file, otherwise it will get overwritten when `make generate` is run 

I suspect this is happening in e2e tests too

Reply:
Uh oh, yes, I totally missed that the CI uses templates under template/test/CI. Updating..